onadata/settings/testing.py

```diff
@@ -1,6 +1,7 @@
 # coding: utf-8
 import os
 
+from fakeredis import FakeConnection, FakeStrictRedis, FakeServer
```

Comment:
Ah, was the support for locking without PostgreSQL exclusively for unit testing?